client/state/reader/start/selectors.js

```diff
@@ -9,6 +13,19 @@ export function isRequestingRecommendations( state ) {
 }
 
 /**
+ * Returns a single recommendation by ID.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Integer}  recommendationId  Recommendation ID
+ * @return {Object} Recommendation
+ */
+export function getRecommendationById( state, recommendationId ) {
+	return find( state.reader.start.items, ( rec ) => {
```

Comment:
i think this can just be `find( state.reader.start.items, { ID: recommendationId } )`